src/pages/BuyGMX/BuyGMX.js

```diff
@@ -1,29 +1,210 @@
 import React, { useCallback } from "react";
 import Footer from "components/Footer/Footer";
 import "./BuyGMX.css";
-
 import { useWeb3React } from "@web3-react/core";
-
-import Synapse from "img/ic_synapse.svg";
-import Multiswap from "img/ic_multiswap.svg";
-import Hop from "img/ic_hop.svg";
-import Banxa from "img/ic_banxa.svg";
-import Binance from "img/ic_binance_logo.svg";
-import avax30Icon from "img/ic_avax_30.svg";
-import gmxArbitrum from "img/ic_gmx_arbitrum.svg";
-import gmxAvax from "img/ic_gmx_avax.svg";
-import ohmArbitrum from "img/ic_olympus_arbitrum.svg";
-
-import { Trans } from "@lingui/macro";
+import { Trans, t } from "@lingui/macro";
 import Button from "components/Common/Button";
-import { ARBITRUM, AVALANCHE } from "config/chains";
+import { ARBITRUM, AVALANCHE, getChainName, getConstant } from "config/chains";
 import { switchNetwork } from "lib/wallets";
 import { useChainId } from "lib/chains";
+import Card from "components/Common/Card";
+import { importImage } from "lib/legacy";
 import ExternalLink from "components/ExternalLink/ExternalLink";
 
+import Banxa from "img/ic_banxa.svg";
+import Uniswap from "img/ic_uni_arbitrum.svg";
+import Traderjoe from "img/ic_traderjoe_avax.png";
+import Bungee from "img/ic_bungee.png";
+import O3 from "img/ic_o3.png";
+import Binance from "img/ic_binance.svg";
+import ohmArbitrum from "img/ic_olympus_arbitrum.svg";
+
+const EXTERNAL_LINKS = {
+  bungee: { [ARBITRUM]: "https://multitx.bungee.exchange/", [AVALANCHE]: "https://multitx.bungee.exchange/" },
+  banxa: {
+    [ARBITRUM]: "https://gmx.banxa.com/?coinType=ETH&fiatType=USD&fiatAmount=500&blockchain=arbitrum",
+    [AVALANCHE]: "https://gmx.banxa.com/?coinType=AVAX&fiatType=USD&fiatAmount=500&blockchain=avalanche",
+  },
+  o3: { [ARBITRUM]: "https://o3swap.com/", [AVALANCHE]: "https://o3swap.com/" },
+  buyGmx: {
+    banxa: {
+      [ARBITRUM]: "https://gmx.banxa.com/?coinType=GMX&fiatType=USD&fiatAmount=500&blockchain=arbitrum",
+      [AVALANCHE]: "https://gmx.banxa.com/?coinType=GMX&fiatType=USD&fiatAmount=500&blockchain=avalanche",
+    },
+    main: {
+      [ARBITRUM]:
+        "https://app.uniswap.org/#/swap?inputCurrency=ETH&outputCurrency=0xfc5A1A6EB076a2C7aD06eD22C90d7E710E35ad0a",
+      [AVALANCHE]: "https://traderjoexyz.com/trade?outputCurrency=0x62edc0692BD897D2295872a9FFCac5425011c661#/",
+    },
+  },
+  nativeNetwork: { [ARBITRUM]: "https://arbitrum.io/", [AVALANCHE]: "https://www.avax.network/" },
+  ftx: { [ARBITRUM]: "https://ftx.com/trade/ETH/USD", [AVALANCHE]: "https://ftx.com/trade/AVAX/USD" },
+};
+
+const TRANSFER_EXCHANGES = [
```

Comment:
the file looks pretty large, maybe move the constants to a separated file? like `src/pages/BuyGMX/constants.ts`?

Reply:
Sure!

Reply:
Done!